### PR TITLE
Recovery mode improvements

### PR DESCRIPTION
**Pull request by bfollington**
Fixes https://github.com/subconsciousnetwork/subconscious/issues/963
Fixes https://github.com/subconsciousnetwork/subconscious/issues/964
Fixes https://github.com/subconsciousnetwork/subconscious/issues/965

- [x] Iterate on wording
    - [ ] What should it say above the "Recovery Phrase" on the form step?
- [x] Error is selectable
- [x] Conceal debug details with `DisclosureGroup`
- [x] Conceal sphere fields with `DisclosureGroup`
    - [x] Auto-expand if this data is missing
- [x] Trim trailing whitespace from pasted values
- [ ] Find cause of hyphenation https://github.com/subconsciousnetwork/subconscious/issues/962

image

image

image

image

### CODE REVIEW

File: xcode/Subconscious/Shared/Components/Common/Recovery/RecoveryModeFormPanelView.swift [hunk 1]
```diff
@@ -19,38 +19,13 @@ struct RecoveryModeFormPanelView: View {
...
-                placeholder: "http://example.com",
-                field: store.state.recoveryGatewayURLField,
-                send: Address.forward(
-                    send: store.send,
-                    tag: RecoveryModeAction.recoveryGatewayURLField
-                ),
-                caption: String(localized: "The URL of your preferred Noosphere gateway"),
-                axis: .vertical
+            Text(
+                "Enter your 24-word recovery phrase to re-download your data."
```
Comment (bfollington) [inline xcode/Subconscious/Shared/Components/Common/Recovery/RecoveryModeFormPanelView.swift:26]: This need to change.

**Comment by gordonbrander** [comment]
@bfollington made some refinements in https://github.com/subconsciousnetwork/subconscious/pull/968

**Review by gordonbrander — CHANGES_REQUESTED** [review]
@bfollington made some refinements in https://github.com/subconsciousnetwork/subconscious/pull/968

**Review by gordonbrander — APPROVED** [review]
Let's merge #968 then LGTM

**Pull request merged**